client/components/QuestionsAndAnswers/QAndAComponents/QuestionSearch.jsx

```diff
@@ -11,15 +11,22 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-export default function QuestionSearch() {
+export default function QuestionSearch({ questions, questionSearch }) {
```

Comment:
It looks like you passed questions as props, but aren't using it anywhere?

Reply:
Good call, I don't think I need that in QuestionSearch, but I'll double check!